pkg/api/admin_users_test.go

```diff
@@ -96,17 +110,9 @@ func TestAdminAPIEndpoint(t *testing.T) {
 	t.Run("When a server admin attempts to enable/disable a nonexistent user", func(t *testing.T) {
 		adminDisableUserScenario(t, "Should return user not found on a POST request", "enable",
 			"/api/admin/users/42/enable", "/api/admin/users/:id/enable", func(sc *scenarioContext) {
-				var userID int64
-				isDisabled := false
-				bus.AddHandler("test", func(ctx context.Context, cmd *models.GetAuthInfoQuery) error {
-					return models.ErrUserNotFound
-				})
-
-				bus.AddHandler("test", func(ctx context.Context, cmd *models.DisableUserCommand) error {
-					userID = cmd.UserId
-					isDisabled = cmd.IsDisabled
-					return models.ErrUserNotFound
-				})
+				store := sc.sqlStore.(*mockstore.SQLStoreMock)
```

Comment:
you can also use the struct "constructor" when building the store instead. it is the same, might seems easier.